lib/types/step_function.ts

```diff
@@ -80,21 +80,35 @@ export interface TaskState extends BaseStateAction {
   HeartbeatSecondsPath?: string;
 }
 
-// https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-map-state.html
+/** https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-map-state.html */
 export interface MapState extends BaseStateAction {
   Type: "Map";
   Iterator: StepFunction;
   ItemsPath: string;
   MaxConcurrency: number;
 }
 
-// https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-parallel-state.html
+/** https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-parallel-state.html */
 export interface ParallelState extends BaseStateAction {
   Type: "Parallel";
   Branches: StepFunction[];
 }
 
-// https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-choice-state.html#amazon-states-language-choice-state-rules
+export interface Catch {
+  ErrorEquals: string[]; // must have at least 1 string.
+  Next: string;
+  ResultPath?: String;
+}
+
+/**https://docs.aws.amazon.com/step-functions/latest/dg/concepts-error-handling.html#error-handling-retrying-after-an-error */
```

Comment:
Looks like we're missing a space here aafter `**`

Reply:
updated comment